docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       - DB_PASSWORD=password
       - KTOR_ENV=dev
       - KTOR_PORT=8080
-      - JAVA_OPTS=-server -Xms256m -Xmx512m -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:+UseStringDeduplication -XX:NativeMemoryTracking=summary -Xrunjdwp:server=y,transport=dt_socket,address=5005,suspend=n
+      - JAVA_OPTS=-server -Xms256m -Xmx512m -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:+UseStringDeduplication -XX:NativeMemoryTracking=summary -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
```

Comment:
Minor update to the debug configs so that we can simply open a remote debugging session in the IDE to the running code. @rmpt this may interest you